lib/bolognese/readers/codemeta_reader.rb

```diff
@@ -81,7 +81,7 @@ def read_codemeta(string: nil, **options)
           "publication_year" => publication_year,
           "descriptions" => meta.fetch("description", nil).present? ? [{ "description" => sanitize(meta.fetch("description")), "descriptionType" => "Abstract" }] : nil,
           "rights_list" => rights_list,
-          "version" => meta.fetch("version", nil),
+          "version_info" => meta.fetch("version", nil),
```

Comment:
[Repeated] I guess this has nothing to do with `rightsidentifier` and its related to another issue. Could you help me to understand, what is this about? I thought all the version swapping was resolved.

Reply:
see below. This is here because I needed to sync the pull request with the master branch.